guides/v2.1/extension-dev-guide/cache/partial-caching/database-caching.md

```diff
@@ -13,29 +13,30 @@ redirect_from:
   -  /guides/v2.2/config-guide/database/database.html
 ---
 
-This topic discusses how to use the Magento 2 database for caching. After you complete these tasks, cached objects are stored in the `cache` and `cache_tag` Magento 2 database tables. Nothing is stored in `var/cache` or `var/page_cache`.
+## Overview of database caching {#mage-cache-db-over}
+This topic discusses how to use the Magento 2 database for caching. After you complete these tasks, cached objects are stored in the `cache` and `cache_tag` Magento 2 database tables. Nothing is stored `var/cache` or `var/page_cache`.
 
 This topic discusses how to set up database caching and how to verify database caching is working. We discuss the following options:
 
 *	Using the `default` cache frontend, in which case you modify `di.xml` only.
 *	Using a custom {% glossarytooltip 0bc9c8bc-de1a-4a06-9c99-a89a29c30645 %}cache{% endglossarytooltip %} frontend, in which case you modify `env.php` only.
 
 <div class="bs-callout bs-callout-warning">
```

Comment:
For notes, please use the following syntax:

```
{:.bs-callout .bs-callout-info}
This is a note.
```

Reply:
Fixed